shared/src/helpers/date/format-date/format-date.helper.ts

```diff
@@ -0,0 +1,18 @@
+const formatDate = (dateStr: string): string => {
```

Comment:
```suggestion
const getFormattedDate = (dateStr: string): string => {
```

Reply:
```tsx
// DateFormat should be an enum

getFormattedDate(date: string, dateFormat: DateFormat): string
```
use library for dates `date-fns` package to format the date